Source/Diagnostics/ReducedDiags/LoadBalanceCosts.cpp

```diff
@@ -124,8 +241,10 @@ void LoadBalanceCosts::WriteToFile (int step) const
         // open tmp file to copy data
         std::string fileTmpName = m_path + m_rd_name + ".tmp." + m_extension;
         std::ofstream ofs(fileTmpName, std::ofstream::out);
+
         // write header row
-        // for each box on each level we saved 6 data fields: [cost, proc, lev, i_low, j_low, k_low])
+        // for each box on each level we saved 7 data fields: [cost, proc, lev, i_low, j_low, k_low, hostname])
+        // (m_nDataFields + 1) below accounts for the 6 Real data fields, then 1 string output to write
```

Comment:
can we assign `(m_nDataFields + 1)` to a temporary variable that we use below? Would make the code a bit easier to read.

Reply:
yes, made the switch.